### PR TITLE
Fix many errors in the TCA extension example

### PR DESCRIPTION
**Pull request by mbrodala**
Also reformat the text for better readability.

Releases: master, 10.4, 9.5, 8.7

### CODE REVIEW

File: Documentation/ExtensionArchitecture/ExtendingTca/Examples/Index.rst [hunk 1]
```diff
@@ -74,20 +74,24 @@ Here's the complete code, taken from file
...
-In the second statement the fields are added to the "types" definition of the
-"fe\_users" table by calling
-:php:`ExtensionManagementUtility::addToAllTCAtypes()`. The first parameter is
-the table name once more. The second parameter is a string, in the syntax as
-used in the :ref:`showitem property of types in TCA
-<t3tca:types-properties-showitem>`. An optional third
-specify the position (:php:`'before'` or :php:`'after'`) in relation to an
-existing field. So you could call optionally::
+In this example the first method call adds fields using
+:php:`ExtensionManagementUtility::addTCAcolumns()`. Parameters:
```
Comment (susannemoog) [inline Documentation/ExtensionArchitecture/ExtendingTca/Examples/Index.rst:78]: should we describe the outcome here (something like "their column configuration is now available, but they are not shown as they are not configured for a type" to make it clearer why/what the next step is about?

Comment (mbrodala) [reply Documentation/ExtensionArchitecture/ExtendingTca/Examples/Index.rst:78]: Yes, good idea.

**Review by linawolf — APPROVED** [review]

**Comment by linawolf** [comment]
@mbrodala can't automatically merge this into 8.7. Please make a separate PR directly into 8.7

**Comment by mbrodala** [comment]
I probably won't have time for this so we can leave it be. The changes are more relevant to newly developed code which is unlikely for older versions.

**Pull request merged**